src/css/graphql/graphql-playground.css

```diff
@@ -339,6 +339,97 @@ div[data-radix-popper-content-wrapper] * {
+    with the class "graphiql-plugin" already provides vertical (y-axis) scrolling.
+    For horizontal (x-axis) scrolling, each GraphQL queries/mutations container will handle its own scroll.
+*/
+.graphql-playground-view #graphiql .docExplorerWrap .graphiql-explorer-root > div {
+    overflow: hidden !important;
+}
+
+/*
+  Adds horizontal (x-axis) scrolling to each GraphQL queries/mutations container.
+*/
+.graphql-playground-view #graphiql .docExplorerWrap .graphiql-explorer-root > div:first-child > div,
+.graphql-playground-view #graphiql .docExplorerWrap .graphiql-explorer-root > div:has(.variable-editor-title) {
+    overflow-x: auto !important;
+}
+
+/* Added a separation line between the container with queries/mutations and the container with the form for adding a new query/mutation. */
+.graphql-playground-view #graphiql .docExplorerWrap .graphiql-explorer-root > div:first-child {
+    border-bottom: 1px solid rgb(214, 214, 214);
+}
+
+/* Removes the top border from the form for adding a new query/mutation,
+   which was used as a visual horizontal separator between the queries/mutations and the form. */
+.graphql-playground-view #graphiql .docExplorerWrap .graphiql-explorer-root > div:last-child .variable-editor-title {
+    border: none !important;
+    padding: 0 !important;
+}
+
+.graphql-playground-view #graphiql .docExplorerWrap .graphiql-explorer-root > div:has(.variable-editor-title) {
+    margin-top: 16px;
+}
+
+/* =========================*/
+/*     Button overrides     */
+/* =========================*/
+.graphql-playground-view #graphiql .docExplorerWrap button.toolbar-button {
+    background-color: var(--base-background-hover-color);
+    border: none;
+    outline: none;
+    cursor: pointer;
+}
+
+.graphql-playground-view #graphiql .docExplorerWrap .variable-editor-title button.toolbar-button,
+.graphql-playground-view #graphiql .docExplorerWrap .graphiql-operation-title-bar button.toolbar-button {
+    height: 100% !important;
+    width: auto !important;
+    aspect-ratio: 1 / 1;
+}
+
+/**
+  Make the span resizable. This allows the icon/text inside the button to become larger on mouseover.
+ */
+.graphql-playground-view #graphiql .docExplorerWrap button.toolbar-button span {
+    display: block;
+}
+
+.graphql-playground-view #graphiql .docExplorerWrap button.toolbar-button:hover span {
+    transform: scale(1.2);
+    color: var(--primary-color) !important;
+}
+
+
+/* Make the button's size match the dropdown in the 'Add New Fragment' form */
+.graphql-playground-view #graphiql .docExplorerWrap .variable-editor-title button.toolbar-button {
+    padding: var(--px-4) var(--px-10);
+}
+
+/* =========================*/
+/*     Dropdown overrides     */
+/* =========================*/
+.graphql-playground-view #graphiql .docExplorerWrap select {
+    font-size: 0.8rem !important;
+    height: calc(2rem - 2px);
+    line-height: 1.25;
+    color: #55595c;
```

Comment:
where this color came from? It's different from ours. Also currently this is overriden runtime from some other value

Reply:
fixed